packages/components/src/presentation-components/common/ContentBuilder.ts

```diff
@@ -139,9 +139,10 @@ class StructMembersAppender implements INestedPropertiesAppender {
     this._members[record.fieldHierarchy.field.name] = record.record;
   }
   public finish(): void {
+    const sortedMembers = Object.entries(this._members).sort();
```

Comment:
In what order is this going to sort? `Object.entries` returns an array of tuples and `sort` orders items in ascending ASCII order 🤔 I think in the best case scenario it's going to be by the first tuple item - the property name, which is not necessarily correct. We want to sort by property label instead.

Reply:
It was sorting based on the property name, changed to sort by label instead.